src/descriptors/power_system_structs.json

```diff
@@ -1771,13 +1771,51 @@
         "name": "states",
         "comment": "Fixed AVR has no states",
         "internal_default": "Vector{Symbol}()",
-        "data_type": "Array{Symbol,1}"
+        "data_type": "Vector{Symbol}"
       },
       {
         "name": "n_states",
         "comment": "Fixed AVR has no states",
+        "null_value": 0,
+        "internal_default": 0,
+        "data_type": "Int64"
+      },
+      {
+        "name": "internal",
+        "comment": "power system internal reference, do not modify",
+        "data_type": "InfrastructureSystemsInternal",
+        "internal_default": "InfrastructureSystemsInternal()"
+      }
+    ],
+    "supertype": "AVR"
+  },
+  {
+    "struct_name": "AVRSimple",
+    "docstring": "Parameters of a simple proportional AVR in the derivative of EMF\ni.e. an integrator controller on EMF",
```

Comment:
I don't think we want to include formatting characters like \n in these strings.  A higher level could do that automatically.  But, I'm open.

Reply:
I was getting issues with the JSON format. What do you suggest?

Reply:
Don't include \n in the string.  If we want line wrapping in the generated docstrings, we can do later in generate_structs.jl (later).

Reply:
let's solve this later.